django_ical/tests/test_feed.py

```diff
@@ -159,6 +198,18 @@ def test_items(self):
         self.assertEquals(calendar.subcomponents[0]['GEO'].to_ical(), "37.386013;-122.082932")
         self.assertEquals(calendar.subcomponents[0]['LAST-MODIFIED'].to_ical(), b'20120502T100000Z')
         self.assertEquals(calendar.subcomponents[0]['ORGANIZER'].to_ical(), b'MAILTO:john.doe@example.com')
+        self.assertEquals(calendar.subcomponents[0]['ATTENDEE'].to_ical(), b'MAILTO:joe.unresponsive@example.com')
+                          # b'ATTENDEE;CN="Joe Unresponsive";CUTYPE=INDIVIDUAL;ROLE=REQ-PARTICIPANT;RSVP=TRUE;PARTSTAT'
```

Comment:
Why are these commented out at the moment?

Reply:
Just trying to get tests to pass first, will add back in once I figure out what I did wrong - if you have any insight, please share.

Reply:
I didn't see anything alarming in the codebase, so I'd try the process of elimination (remove lines until they pass) first to find out the malfunctioning things.

Reply:
Did you check the types and semantics of these items? Is `ATTENDEE` field a bytes item or something else?

Reply:
> Did you check the types and semantics of these items? Is `ATTENDEE` field a bytes item or something else?

Sorry, I did not check- I ran with the assumption that ATTENDEE & ORGANIZER would be
tested in the same manner.